### PR TITLE
Styles landing

### PR DESCRIPTION
**Pull request by dharness**
I wrote a lot of styles for the landing page. It is basically functional at this point

### CODE REVIEW

**Comment by nnur** [comment]
LGTM

File: landing/app/index.html [hunk 1]
```diff
@@ -2,41 +2,88 @@
...
+    <div class="bablot-contact-panel">
+    <div class="bablot-contact">
+      <div class="bablot-contact--header">We don't bite</div>
+        <div class="bablot-contact--phone">
+        <img src="assets/phone.png">
+        226 378 8121
+      </div>
+      <div class="bablot-contact--email">
+        <img src="assets/email.png">
+        yourfriends@bablot.co
```
Comment (moskalyk) [inline landing/app/index.html]: add in a mailto:

File: landing/app/index.html [hunk 2]
```diff
@@ -2,41 +2,88 @@
...
+          <p>If you have any questions about finding a place, just ask Sir Bablot!</p>
+        </div>
+      </div>
+    </div>
+    <div class="bablot-contact-panel">
+    <div class="bablot-contact">
+      <div class="bablot-contact--header">We don't bite</div>
+        <div class="bablot-contact--phone">
+        <img src="assets/phone.png">
+        226 378 8121
```
Comment (moskalyk) [inline landing/app/index.html]: add in phone me plz

File: landing/app/main.js [hunk 3]
```diff
@@ -1,6 +1,61 @@
...
+
+    contactPanel.style.display = 'block';
+      window.getComputedStyle(contactPanel).display;
+    contactPanel.style.opacity = '1';
+
+    aboutPanel.style.display = 'none';
+    aboutPanel.style.opacity = '0';
+
+    intro.style.opacity = '0';
+    // crown.style.opacity = '0';
```
Comment (moskalyk) [inline landing/app/main.js]: do we need this anymore?

Comment (dharness) [inline landing/app/main.js]: Removed

File: landing/app/index.html (hunk 2 above)
Comment (dharness) [inline landing/app/index.html]: doen

File: landing/app/index.html (hunk 1 above)
Comment (dharness) [inline landing/app/index.html]: done

**Comment by moskalyk** [comment]
LGTM

**Pull request merged**